include/internal/ktls.h

```diff
@@ -195,12 +195,10 @@ static ossl_inline ossl_ssize_t ktls_sendfile(int s, int fd, off_t off,
     off_t sbytes;
     int ret;
 
+    sbytes = 0;
```

Comment:
Please just initialise this in the declaration above

Reply:
Fixed in the new revision.